_data/simple-icons.json

```diff
@@ -3410,6 +3410,16 @@
             "hex": "BE2323",
             "source": "https://www.ebox.ca"
         },
+        {
+            "title": "Eclipse Adoptium",
+            "hex": "FF1464",
+            "source": "https://adoptium.net",
+            "guidelines": "https://www.eclipse.org/org/artwork/guidelines/adoptium-brand-guidelines.pdf",
+            "license": {
+                "type": "EPL-2.0",
```

Comment:
```suggestion
```
The icon is not provided under EPL-2.0 License. If I understand correctly it is the software that is provided under the aforementioned license.